workflow/common/util.go

```diff
@@ -300,19 +301,13 @@ func ProcessArgs(tmpl *wfv1.Template, args wfv1.ArgumentsProvider, globalParams,
 }
 
 // SubstituteParams returns a new copy of the template with global, pod, and input parameters substituted
-func SubstituteParams(tmpl *wfv1.Template, globalParams, localParams map[string]string) (*wfv1.Template, error) {
+func SubstituteParams(tmpl *wfv1.Template, globalParams, localParams params.Params) (*wfv1.Template, error) {
```

Comment:
When we move the util file under `common`, we won't need the import prefix anymore.